Dockerfile

```diff
@@ -11,4 +11,6 @@ RUN mkdir /app/log
 
 COPY . /app/
 
+RUN cp /app/config/site.yml.tmpl /app/config/site.yml
```

Comment:
I'm not convinced that copying the `site.yml` template into the Docker image is the right thing to do here. Since `site.yml` contains a bunch of customization that a user would need to do to install Tracks, I'd rather leave it out by default.

Reply:
True, I'll do that, I was just aiming to have a one command only to start, but it's true what you are saying.

What about copying it only if it's not present?